docs/source/architecture/partitioning.rst

```diff
@@ -0,0 +1,33 @@
+===========================================
+Snuba Data Partitioning (under development)
+===========================================
+
+*This feature is under active development and is subject to change*
+
+To support a higher volume of data, we are building out support for
+datasets and storages that span multiple ClickHouse clusters with the same
```

Comment:
Just ClickHouse clusters? How about mentioning Kafka clusters and potentially Redises (tbc) and Postgreses (well, that one would be part of Sentry but still...). I was under the impression the main purpose of this common partitioning scheme we are doing is to facilitate a single approach over multiple systems and not just ClickHouse though.

Reply:
that's a good call, I can update the description